keras_cv/visualization/draw_bounding_boxes.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2022 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+def draw_bounding_boxes(images, bounding_boxes, colors):
```

Comment:
This does not seem to be used within the PR, maybe remove it?

Reply:
Oops - did not mean to put that in there.